Binary files a/._.eslintrc.json and b/._.eslintrc.json differ
Binary files a/._.vscode and b/._.vscode differ
Binary files a/._CODE_OF_CONDUCT.md and b/._CODE_OF_CONDUCT.md differ
Binary files a/._CONTRIBUTING.md and b/._CONTRIBUTING.md differ
Binary files a/._LICENSE and b/._LICENSE differ
Binary files a/._README.md and b/._README.md differ
Binary files a/._SECURITY.md and b/._SECURITY.md differ
Binary files a/._bin and b/._bin differ
Binary files a/._contrib and b/._contrib differ
Binary files a/._copyparty and b/._copyparty differ
Binary files a/._docs and b/._docs differ
Binary files a/._flake.lock and b/._flake.lock differ
Binary files a/._flake.nix and b/._flake.nix differ
Binary files a/._pyproject.toml and b/._pyproject.toml differ
Binary files a/._scripts and b/._scripts differ
Binary files a/._setup.py and b/._setup.py differ
Binary files a/._srv and b/._srv differ
Binary files a/._tests and b/._tests differ
Binary files a/.vscode/._launch.json and b/.vscode/._launch.json differ
Binary files a/.vscode/._launch.py and b/.vscode/._launch.py differ
Binary files a/.vscode/._settings.json and b/.vscode/._settings.json differ
Binary files a/.vscode/._tasks.json and b/.vscode/._tasks.json differ
Binary files a/bin/._README.md and b/bin/._README.md differ
Binary files a/bin/._dbtool.py and b/bin/._dbtool.py differ
Binary files a/bin/._handlers and b/bin/._handlers differ
Binary files a/bin/._hooks and b/bin/._hooks differ
Binary files a/bin/._mtag and b/bin/._mtag differ
Binary files a/bin/._partyfuse-streaming.py and b/bin/._partyfuse-streaming.py differ
Binary files a/bin/._partyfuse.py and b/bin/._partyfuse.py differ
Binary files a/bin/._partyfuse2.py and b/bin/._partyfuse2.py differ
Binary files a/bin/._partyjournal.py and b/bin/._partyjournal.py differ
Binary files a/bin/._prisonparty.sh and b/bin/._prisonparty.sh differ
Binary files a/bin/._u2c.py and b/bin/._u2c.py differ
Binary files a/bin/._unforget.py and b/bin/._unforget.py differ
Binary files a/bin/._up2k.sh and b/bin/._up2k.sh differ
Binary files a/bin/handlers/._README.md and b/bin/handlers/._README.md differ
Binary files a/bin/handlers/._caching-proxy.py and b/bin/handlers/._caching-proxy.py differ
Binary files a/bin/handlers/._ip-ok.py and b/bin/handlers/._ip-ok.py differ
Binary files a/bin/handlers/._never404.py and b/bin/handlers/._never404.py differ
Binary files a/bin/handlers/._nooo.py and b/bin/handlers/._nooo.py differ
Binary files a/bin/handlers/._sorry.py and b/bin/handlers/._sorry.py differ
Binary files a/bin/hooks/._README.md and b/bin/hooks/._README.md differ
Binary files a/bin/hooks/._discord-announce.py and b/bin/hooks/._discord-announce.py differ
Binary files a/bin/hooks/._image-noexif.py and b/bin/hooks/._image-noexif.py differ
Binary files a/bin/hooks/._into-the-cache-it-goes.py and b/bin/hooks/._into-the-cache-it-goes.py differ
Binary files a/bin/hooks/._msg-log.py and b/bin/hooks/._msg-log.py differ
Binary files a/bin/hooks/._notify.py and b/bin/hooks/._notify.py differ
Binary files a/bin/hooks/._notify2.py and b/bin/hooks/._notify2.py differ
Binary files a/bin/hooks/._qbittorrent-magnet.py and b/bin/hooks/._qbittorrent-magnet.py differ
Binary files a/bin/hooks/._reject-extension.py and b/bin/hooks/._reject-extension.py differ
Binary files a/bin/hooks/._reject-mimetype.py and b/bin/hooks/._reject-mimetype.py differ
Binary files a/bin/hooks/._reloc-by-ext.py and b/bin/hooks/._reloc-by-ext.py differ
Binary files a/bin/hooks/._wget.py and b/bin/hooks/._wget.py differ
Binary files a/bin/hooks/._xiu-sha.py and b/bin/hooks/._xiu-sha.py differ
Binary files a/bin/hooks/._xiu.py and b/bin/hooks/._xiu.py differ
Binary files a/bin/mtag/._README.md and b/bin/mtag/._README.md differ
Binary files a/bin/mtag/._audio-bpm.py and b/bin/mtag/._audio-bpm.py differ
Binary files a/bin/mtag/._audio-key-slicing.py and b/bin/mtag/._audio-key-slicing.py differ
Binary files a/bin/mtag/._audio-key.py and b/bin/mtag/._audio-key.py differ
Binary files a/bin/mtag/._cksum.py and b/bin/mtag/._cksum.py differ
Binary files a/bin/mtag/._exe.py and b/bin/mtag/._exe.py differ
Binary files a/bin/mtag/._file-ext.py and b/bin/mtag/._file-ext.py differ
Binary files a/bin/mtag/._guestbook-read.py and b/bin/mtag/._guestbook-read.py differ
Binary files a/bin/mtag/._guestbook.py and b/bin/mtag/._guestbook.py differ
Binary files a/bin/mtag/._image-noexif.py and b/bin/mtag/._image-noexif.py differ
Binary files a/bin/mtag/._install-deps.sh and b/bin/mtag/._install-deps.sh differ
Binary files a/bin/mtag/._media-hash.py and b/bin/mtag/._media-hash.py differ
Binary files a/bin/mtag/._mousepad.py and b/bin/mtag/._mousepad.py differ
Binary files a/bin/mtag/._rclone-upload.py and b/bin/mtag/._rclone-upload.py differ
Binary files a/bin/mtag/._res and b/bin/mtag/._res differ
Binary files a/bin/mtag/._sleep.py and b/bin/mtag/._sleep.py differ
Binary files a/bin/mtag/._very-bad-idea.py and b/bin/mtag/._very-bad-idea.py differ
Binary files a/bin/mtag/._vidchk.py and b/bin/mtag/._vidchk.py differ
Binary files a/bin/mtag/._wget.py and b/bin/mtag/._wget.py differ
Binary files a/bin/mtag/._yt-ipr.py and b/bin/mtag/._yt-ipr.py differ
Binary files a/bin/mtag/res/._twitter-unmute.user.js and b/bin/mtag/res/._twitter-unmute.user.js differ
Binary files a/bin/mtag/res/._yt-ipr.conf and b/bin/mtag/res/._yt-ipr.conf differ
Binary files a/bin/mtag/res/._yt-ipr.user.js and b/bin/mtag/res/._yt-ipr.user.js differ
Binary files a/contrib/._README.md and b/contrib/._README.md differ
Binary files a/contrib/._apache and b/contrib/._apache differ
Binary files a/contrib/._cfssl.sh and b/contrib/._cfssl.sh differ
Binary files a/contrib/._copyparty.bat and b/contrib/._copyparty.bat differ
Binary files a/contrib/._explorer-nothumbs-nofoldertypes.reg and b/contrib/._explorer-nothumbs-nofoldertypes.reg differ
Binary files a/contrib/._flameshot.sh and b/contrib/._flameshot.sh differ
Binary files a/contrib/._index.html and b/contrib/._index.html differ
Binary files a/contrib/._ios and b/contrib/._ios differ
Binary files a/contrib/._media-osd-bgone.ps1 and b/contrib/._media-osd-bgone.ps1 differ
Binary files a/contrib/._nginx and b/contrib/._nginx differ
Binary files a/contrib/._nixos and b/contrib/._nixos differ
Binary files a/contrib/._openrc and b/contrib/._openrc differ
Binary files a/contrib/._package and b/contrib/._package differ
Binary files a/contrib/._plugins and b/contrib/._plugins differ
Binary files a/contrib/._rc and b/contrib/._rc differ
Binary files a/contrib/._send-to-cpp.contextlet.json and b/contrib/._send-to-cpp.contextlet.json differ
Binary files a/contrib/._sharex.sxcu and b/contrib/._sharex.sxcu differ
Binary files a/contrib/._sharex12.sxcu and b/contrib/._sharex12.sxcu differ
Binary files a/contrib/._systemd and b/contrib/._systemd differ
Binary files a/contrib/._themes and b/contrib/._themes differ
Binary files a/contrib/._webdav-cfg.bat and b/contrib/._webdav-cfg.bat differ
Binary files a/contrib/._windows and b/contrib/._windows differ
Binary files a/contrib/apache/._copyparty.conf and b/contrib/apache/._copyparty.conf differ
Binary files a/contrib/ios/._upload-to-copyparty.shortcut and b/contrib/ios/._upload-to-copyparty.shortcut differ
Binary files a/contrib/nginx/._copyparty.conf and b/contrib/nginx/._copyparty.conf differ
Binary files a/contrib/nixos/._modules and b/contrib/nixos/._modules differ
Binary files a/contrib/nixos/modules/._copyparty.nix and b/contrib/nixos/modules/._copyparty.nix differ
Binary files a/contrib/openrc/._copyparty and b/contrib/openrc/._copyparty differ
Binary files a/contrib/package/._arch and b/contrib/package/._arch differ
Binary files a/contrib/package/._nix and b/contrib/package/._nix differ
Binary files a/contrib/package/arch/._PKGBUILD and b/contrib/package/arch/._PKGBUILD differ
Binary files a/contrib/package/arch/._copyparty.conf and b/contrib/package/arch/._copyparty.conf differ
Binary files a/contrib/package/arch/._copyparty.service and b/contrib/package/arch/._copyparty.service differ
Binary files a/contrib/package/arch/._index.md and b/contrib/package/arch/._index.md differ
Binary files a/contrib/package/arch/._prisonparty.service and b/contrib/package/arch/._prisonparty.service differ
Binary files a/contrib/package/nix/._copyparty and b/contrib/package/nix/._copyparty differ
Binary files a/contrib/package/nix/copyparty/._default.nix and b/contrib/package/nix/copyparty/._default.nix differ
Binary files a/contrib/package/nix/copyparty/._pin.json and b/contrib/package/nix/copyparty/._pin.json differ
Binary files a/contrib/package/nix/copyparty/._update.py and b/contrib/package/nix/copyparty/._update.py differ
Binary files a/contrib/plugins/._README.md and b/contrib/plugins/._README.md differ
Binary files a/contrib/plugins/._banner.js and b/contrib/plugins/._banner.js differ
Binary files a/contrib/plugins/._browser-icons.css and b/contrib/plugins/._browser-icons.css differ
Binary files a/contrib/plugins/._meadup.js and b/contrib/plugins/._meadup.js differ
Binary files a/contrib/plugins/._minimal-up2k.html and b/contrib/plugins/._minimal-up2k.html differ
Binary files a/contrib/plugins/._minimal-up2k.js and b/contrib/plugins/._minimal-up2k.js differ
Binary files a/contrib/plugins/._rave.js and b/contrib/plugins/._rave.js differ
Binary files a/contrib/plugins/._up2k-hook-ytid.js and b/contrib/plugins/._up2k-hook-ytid.js differ
Binary files a/contrib/plugins/._up2k-hooks.js and b/contrib/plugins/._up2k-hooks.js differ
Binary files a/contrib/rc/._copyparty and b/contrib/rc/._copyparty differ
Binary files a/contrib/systemd/._cfssl.service and b/contrib/systemd/._cfssl.service differ
Binary files a/contrib/systemd/._copyparty.conf and b/contrib/systemd/._copyparty.conf differ
Binary files a/contrib/systemd/._copyparty.service and b/contrib/systemd/._copyparty.service differ
Binary files a/contrib/systemd/._prisonparty.service and b/contrib/systemd/._prisonparty.service differ
Binary files a/contrib/themes/._bsod.css and b/contrib/themes/._bsod.css differ
Binary files a/contrib/themes/._bsod.png and b/contrib/themes/._bsod.png differ
Binary files a/contrib/windows/._copyparty-ctmp.bat and b/contrib/windows/._copyparty-ctmp.bat differ
Binary files a/copyparty/.___init__.py and b/copyparty/.___init__.py differ
Binary files a/copyparty/.___main__.py and b/copyparty/.___main__.py differ
Binary files a/copyparty/.___version__.py and b/copyparty/.___version__.py differ
Binary files a/copyparty/._authsrv.py and b/copyparty/._authsrv.py differ
Binary files a/copyparty/._bos and b/copyparty/._bos differ
Binary files a/copyparty/._broker_mp.py and b/copyparty/._broker_mp.py differ
Binary files a/copyparty/._broker_mpw.py and b/copyparty/._broker_mpw.py differ
Binary files a/copyparty/._broker_thr.py and b/copyparty/._broker_thr.py differ
Binary files a/copyparty/._broker_util.py and b/copyparty/._broker_util.py differ
Binary files a/copyparty/._cert.py and b/copyparty/._cert.py differ
Binary files a/copyparty/._cfg.py and b/copyparty/._cfg.py differ
Binary files a/copyparty/._dxml.py and b/copyparty/._dxml.py differ
Binary files a/copyparty/._fsutil.py and b/copyparty/._fsutil.py differ
Binary files a/copyparty/._ftpd.py and b/copyparty/._ftpd.py differ
Binary files a/copyparty/._httpcli.py and b/copyparty/._httpcli.py differ
Binary files a/copyparty/._httpconn.py and b/copyparty/._httpconn.py differ
Binary files a/copyparty/._httpsrv.py and b/copyparty/._httpsrv.py differ
Binary files a/copyparty/._ico.py and b/copyparty/._ico.py differ
Binary files a/copyparty/._mdns.py and b/copyparty/._mdns.py differ
Binary files a/copyparty/._metrics.py and b/copyparty/._metrics.py differ
Binary files a/copyparty/._mtag.py and b/copyparty/._mtag.py differ
Binary files a/copyparty/._multicast.py and b/copyparty/._multicast.py differ
Binary files a/copyparty/._pwhash.py and b/copyparty/._pwhash.py differ
Binary files a/copyparty/._res and b/copyparty/._res differ
Binary files a/copyparty/._smbd.py and b/copyparty/._smbd.py differ
Binary files a/copyparty/._ssdp.py and b/copyparty/._ssdp.py differ
Binary files a/copyparty/._star.py and b/copyparty/._star.py differ
Binary files a/copyparty/._stolen and b/copyparty/._stolen differ
Binary files a/copyparty/._sutil.py and b/copyparty/._sutil.py differ
Binary files a/copyparty/._svchub.py and b/copyparty/._svchub.py differ
Binary files a/copyparty/._szip.py and b/copyparty/._szip.py differ
Binary files a/copyparty/._tcpsrv.py and b/copyparty/._tcpsrv.py differ
Binary files a/copyparty/._tftpd.py and b/copyparty/._tftpd.py differ
Binary files a/copyparty/._th_cli.py and b/copyparty/._th_cli.py differ
Binary files a/copyparty/._th_srv.py and b/copyparty/._th_srv.py differ
Binary files a/copyparty/._u2idx.py and b/copyparty/._u2idx.py differ
Binary files a/copyparty/._up2k.py and b/copyparty/._up2k.py differ
Binary files a/copyparty/._util.py and b/copyparty/._util.py differ
Binary files a/copyparty/._web and b/copyparty/._web differ
Binary files a/copyparty/bos/.___init__.py and b/copyparty/bos/.___init__.py differ
Binary files a/copyparty/bos/._bos.py and b/copyparty/bos/._bos.py differ
Binary files a/copyparty/bos/._path.py and b/copyparty/bos/._path.py differ
Binary files a/copyparty/res/.___init__.py and b/copyparty/res/.___init__.py differ
Binary files a/copyparty/res/._insecure.pem and b/copyparty/res/._insecure.pem differ
Binary files a/copyparty/stolen/.___init__.py and b/copyparty/stolen/.___init__.py differ
Binary files a/copyparty/stolen/._dnslib and b/copyparty/stolen/._dnslib differ
Binary files a/copyparty/stolen/._ifaddr and b/copyparty/stolen/._ifaddr differ
Binary files a/copyparty/stolen/._qrcodegen.py and b/copyparty/stolen/._qrcodegen.py differ
Binary files a/copyparty/stolen/._surrogateescape.py and b/copyparty/stolen/._surrogateescape.py differ
Binary files a/copyparty/stolen/dnslib/._README.md and b/copyparty/stolen/dnslib/._README.md differ
Binary files a/copyparty/stolen/dnslib/.___init__.py and b/copyparty/stolen/dnslib/.___init__.py differ
Binary files a/copyparty/stolen/dnslib/._bimap.py and b/copyparty/stolen/dnslib/._bimap.py differ
Binary files a/copyparty/stolen/dnslib/._bit.py and b/copyparty/stolen/dnslib/._bit.py differ
Binary files a/copyparty/stolen/dnslib/._buffer.py and b/copyparty/stolen/dnslib/._buffer.py differ
Binary files a/copyparty/stolen/dnslib/._dns.py and b/copyparty/stolen/dnslib/._dns.py differ
Binary files a/copyparty/stolen/dnslib/._label.py and b/copyparty/stolen/dnslib/._label.py differ
Binary files a/copyparty/stolen/dnslib/._lex.py and b/copyparty/stolen/dnslib/._lex.py differ
Binary files a/copyparty/stolen/dnslib/._ranges.py and b/copyparty/stolen/dnslib/._ranges.py differ
Binary files a/copyparty/stolen/ifaddr/._README.md and b/copyparty/stolen/ifaddr/._README.md differ
Binary files a/copyparty/stolen/ifaddr/.___init__.py and b/copyparty/stolen/ifaddr/.___init__.py differ
Binary files a/copyparty/stolen/ifaddr/.__posix.py and b/copyparty/stolen/ifaddr/.__posix.py differ
Binary files a/copyparty/stolen/ifaddr/.__shared.py and b/copyparty/stolen/ifaddr/.__shared.py differ
Binary files a/copyparty/stolen/ifaddr/.__win32.py and b/copyparty/stolen/ifaddr/.__win32.py differ
Binary files a/copyparty/web/._Makefile and b/copyparty/web/._Makefile differ
Binary files a/copyparty/web/._a and b/copyparty/web/._a differ
Binary files a/copyparty/web/._baguettebox.js and b/copyparty/web/._baguettebox.js differ
Binary files a/copyparty/web/._browser.css and b/copyparty/web/._browser.css differ
Binary files a/copyparty/web/._browser.html and b/copyparty/web/._browser.html differ
Binary files a/copyparty/web/._browser.js and b/copyparty/web/._browser.js differ
Binary files a/copyparty/web/._browser2.html and b/copyparty/web/._browser2.html differ
Binary files a/copyparty/web/._cf.html and b/copyparty/web/._cf.html differ
Binary files a/copyparty/web/._copyparty.gif and b/copyparty/web/._copyparty.gif differ
Binary files a/copyparty/web/._dbg-audio.js and b/copyparty/web/._dbg-audio.js differ
Binary files a/copyparty/web/._dd and b/copyparty/web/._dd differ
Binary files a/copyparty/web/._deps and b/copyparty/web/._deps differ
Binary files a/copyparty/web/._md.css and b/copyparty/web/._md.css differ
Binary files a/copyparty/web/._md.html and b/copyparty/web/._md.html differ
Binary files a/copyparty/web/._md.js and b/copyparty/web/._md.js differ
Binary files a/copyparty/web/._md2.css and b/copyparty/web/._md2.css differ
Binary files a/copyparty/web/._md2.js and b/copyparty/web/._md2.js differ
Binary files a/copyparty/web/._mde.css and b/copyparty/web/._mde.css differ
Binary files a/copyparty/web/._mde.html and b/copyparty/web/._mde.html differ
Binary files a/copyparty/web/._mde.js and b/copyparty/web/._mde.js differ
Binary files a/copyparty/web/._msg.css and b/copyparty/web/._msg.css differ
Binary files a/copyparty/web/._msg.html and b/copyparty/web/._msg.html differ
Binary files a/copyparty/web/._shares.css and b/copyparty/web/._shares.css differ
Binary files a/copyparty/web/._shares.html and b/copyparty/web/._shares.html differ
Binary files a/copyparty/web/._shares.js and b/copyparty/web/._shares.js differ
Binary files a/copyparty/web/._splash.css and b/copyparty/web/._splash.css differ
Binary files a/copyparty/web/._splash.html and b/copyparty/web/._splash.html differ
Binary files a/copyparty/web/._splash.js and b/copyparty/web/._splash.js differ
Binary files a/copyparty/web/._svcs.html and b/copyparty/web/._svcs.html differ
Binary files a/copyparty/web/._svcs.js and b/copyparty/web/._svcs.js differ
Binary files a/copyparty/web/._ui.css and b/copyparty/web/._ui.css differ
Binary files a/copyparty/web/._up2k.js and b/copyparty/web/._up2k.js differ
Binary files a/copyparty/web/._util.js and b/copyparty/web/._util.js differ
Binary files a/copyparty/web/._w.hash.js and b/copyparty/web/._w.hash.js differ
Binary files a/copyparty/web/a/.___init__.py and b/copyparty/web/a/.___init__.py differ
Binary files a/copyparty/web/a/._partyfuse.py and b/copyparty/web/a/._partyfuse.py differ
Binary files a/copyparty/web/a/._u2c.py and b/copyparty/web/a/._u2c.py differ
Binary files a/copyparty/web/a/._webdav-cfg.bat and b/copyparty/web/a/._webdav-cfg.bat differ
Binary files a/copyparty/web/dd/._2.png and b/copyparty/web/dd/._2.png differ
Binary files a/copyparty/web/dd/._3.png and b/copyparty/web/dd/._3.png differ
Binary files a/copyparty/web/dd/._4.png and b/copyparty/web/dd/._4.png differ
Binary files a/copyparty/web/dd/._5.png and b/copyparty/web/dd/._5.png differ
Binary files a/copyparty/web/dd/.___init__.py and b/copyparty/web/dd/.___init__.py differ
Binary files a/copyparty/web/deps/.___init__.py and b/copyparty/web/deps/.___init__.py differ
Binary files a/docs/._README.md and b/docs/._README.md differ
Binary files a/docs/._TODO.md and b/docs/._TODO.md differ
Binary files a/docs/._biquad.html and b/docs/._biquad.html differ
Binary files a/docs/._bufsize.txt and b/docs/._bufsize.txt differ
Binary files a/docs/._changelog.md and b/docs/._changelog.md differ
Binary files a/docs/._copyparty.d and b/docs/._copyparty.d differ
Binary files a/docs/._cursed-usecases and b/docs/._cursed-usecases differ
Binary files a/docs/._design.txt and b/docs/._design.txt differ
Binary files a/docs/._devnotes.md and b/docs/._devnotes.md differ
Binary files a/docs/._example.conf and b/docs/._example.conf differ
Binary files a/docs/._example2.conf and b/docs/._example2.conf differ
Binary files a/docs/._examples and b/docs/._examples differ
Binary files a/docs/._hls.html and b/docs/._hls.html differ
Binary files a/docs/._idp.md and b/docs/._idp.md differ
Binary files a/docs/._lics.txt and b/docs/._lics.txt differ
Binary files a/docs/._logo.svg and b/docs/._logo.svg differ
Binary files a/docs/._multisearch.html and b/docs/._multisearch.html differ
Binary files a/docs/._music-analysis.sh and b/docs/._music-analysis.sh differ
Binary files a/docs/._notes.bat and b/docs/._notes.bat differ
Binary files a/docs/._notes.md and b/docs/._notes.md differ
Binary files a/docs/._notes.sh and b/docs/._notes.sh differ
Binary files a/docs/._nuitka.txt and b/docs/._nuitka.txt differ
Binary files a/docs/._pretend-youre-qnap.patch and b/docs/._pretend-youre-qnap.patch differ
Binary files a/docs/._protocol-reference.sh and b/docs/._protocol-reference.sh differ
Binary files a/docs/._rclone.md and b/docs/._rclone.md differ
Binary files a/docs/._rice and b/docs/._rice differ
Binary files a/docs/._tcp-debug.sh and b/docs/._tcp-debug.sh differ
Binary files a/docs/._unirange.py and b/docs/._unirange.py differ
Binary files a/docs/._up2k.txt and b/docs/._up2k.txt differ
Binary files a/docs/._versus.md and b/docs/._versus.md differ
Binary files a/docs/._xff.md and b/docs/._xff.md differ
Binary files a/docs/copyparty.d/._foo and b/docs/copyparty.d/._foo differ
Binary files a/docs/copyparty.d/._some.conf and b/docs/copyparty.d/._some.conf differ
Binary files a/docs/copyparty.d/foo/._another.conf and b/docs/copyparty.d/foo/._another.conf differ
Binary files a/docs/copyparty.d/foo/._sibling.conf and b/docs/copyparty.d/foo/._sibling.conf differ
Binary files a/docs/cursed-usecases/._README.md and b/docs/cursed-usecases/._README.md differ
Binary files a/docs/examples/._README.md and b/docs/examples/._README.md differ
Binary files a/docs/examples/._docker and b/docs/examples/._docker differ
Binary files a/docs/examples/._windows.md and b/docs/examples/._windows.md differ
Binary files a/docs/examples/docker/._basic-docker-compose and b/docs/examples/docker/._basic-docker-compose differ
Binary files a/docs/examples/docker/._idp and b/docs/examples/docker/._idp differ
Binary files a/docs/examples/docker/._idp-authelia-traefik and b/docs/examples/docker/._idp-authelia-traefik differ
Binary files a/docs/examples/docker/._idp-authentik-traefik and b/docs/examples/docker/._idp-authentik-traefik differ
Binary files a/docs/examples/docker/._portainer.md and b/docs/examples/docker/._portainer.md differ
Binary files a/docs/examples/docker/basic-docker-compose/._copyparty.conf and b/docs/examples/docker/basic-docker-compose/._copyparty.conf differ
Binary files a/docs/examples/docker/basic-docker-compose/._docker-compose.yml and b/docs/examples/docker/basic-docker-compose/._docker-compose.yml differ
Binary files a/docs/examples/docker/idp/._copyparty.conf and b/docs/examples/docker/idp/._copyparty.conf differ
Binary files a/docs/examples/docker/idp-authelia-traefik/._README.md and b/docs/examples/docker/idp-authelia-traefik/._README.md differ
Binary files a/docs/examples/docker/idp-authelia-traefik/._authelia and b/docs/examples/docker/idp-authelia-traefik/._authelia differ
Binary files a/docs/examples/docker/idp-authelia-traefik/._cpp and b/docs/examples/docker/idp-authelia-traefik/._cpp differ
Binary files a/docs/examples/docker/idp-authelia-traefik/._docker-compose.yml and b/docs/examples/docker/idp-authelia-traefik/._docker-compose.yml differ
Binary files a/docs/examples/docker/idp-authelia-traefik/authelia/._configuration.yml and b/docs/examples/docker/idp-authelia-traefik/authelia/._configuration.yml differ
Binary files a/docs/examples/docker/idp-authelia-traefik/authelia/._users_database.yml and b/docs/examples/docker/idp-authelia-traefik/authelia/._users_database.yml differ
Binary files a/docs/examples/docker/idp-authelia-traefik/cpp/._copyparty.conf and b/docs/examples/docker/idp-authelia-traefik/cpp/._copyparty.conf differ
Binary files a/docs/examples/docker/idp-authentik-traefik/._README.md and b/docs/examples/docker/idp-authentik-traefik/._README.md differ
Binary files a/docs/examples/docker/idp-authentik-traefik/._based-on and b/docs/examples/docker/idp-authentik-traefik/._based-on differ
Binary files a/docs/examples/docker/idp-authentik-traefik/._cpp and b/docs/examples/docker/idp-authentik-traefik/._cpp differ
Binary files a/docs/examples/docker/idp-authentik-traefik/._docker-compose.yml and b/docs/examples/docker/idp-authentik-traefik/._docker-compose.yml differ
Binary files a/docs/examples/docker/idp-authentik-traefik/based-on/._docker-compose-authentik.yml and b/docs/examples/docker/idp-authentik-traefik/based-on/._docker-compose-authentik.yml differ
Binary files a/docs/examples/docker/idp-authentik-traefik/based-on/._docker-compose-traefik.yml and b/docs/examples/docker/idp-authentik-traefik/based-on/._docker-compose-traefik.yml differ
Binary files a/docs/examples/docker/idp-authentik-traefik/cpp/._copyparty.conf and b/docs/examples/docker/idp-authentik-traefik/cpp/._copyparty.conf differ
Binary files a/docs/rice/._README.md and b/docs/rice/._README.md differ
Binary files a/scripts/._bench and b/scripts/._bench differ
Binary files a/scripts/._copyparty-android.sh and b/scripts/._copyparty-android.sh differ
Binary files a/scripts/._copyparty-repack.sh and b/scripts/._copyparty-repack.sh differ
Binary files a/scripts/._deps-docker and b/scripts/._deps-docker differ
Binary files a/scripts/._docker and b/scripts/._docker differ
Binary files a/scripts/._fusefuzz.py and b/scripts/._fusefuzz.py differ
Binary files a/scripts/._genlic.sh and b/scripts/._genlic.sh differ
Binary files a/scripts/._help2html.py and b/scripts/._help2html.py differ
Binary files a/scripts/._help2txt.sh and b/scripts/._help2txt.sh differ
Binary files a/scripts/._install-githooks.sh and b/scripts/._install-githooks.sh differ
Binary files a/scripts/._logpack.sh and b/scripts/._logpack.sh differ
Binary files a/scripts/._make-pypi-release.sh and b/scripts/._make-pypi-release.sh differ
Binary files a/scripts/._make-pyz.sh and b/scripts/._make-pyz.sh differ
Binary files a/scripts/._make-sfx.sh and b/scripts/._make-sfx.sh differ
Binary files a/scripts/._make-tgz-release.sh and b/scripts/._make-tgz-release.sh differ
Binary files a/scripts/._prep.sh and b/scripts/._prep.sh differ
Binary files a/scripts/._profile.py and b/scripts/._profile.py differ
Binary files a/scripts/._py2 and b/scripts/._py2 differ
Binary files a/scripts/._pyinstaller and b/scripts/._pyinstaller differ
Binary files a/scripts/._rls.sh and b/scripts/._rls.sh differ
Binary files a/scripts/._run-tests.sh and b/scripts/._run-tests.sh differ
Binary files a/scripts/._sfx.ls and b/scripts/._sfx.ls differ
Binary files a/scripts/._sfx.py and b/scripts/._sfx.py differ
Binary files a/scripts/._sfx.sh and b/scripts/._sfx.sh differ
Binary files a/scripts/._speedtest-fs.py and b/scripts/._speedtest-fs.py differ
Binary files a/scripts/._strip_hints and b/scripts/._strip_hints differ
Binary files a/scripts/._test and b/scripts/._test differ
Binary files a/scripts/._tl.js and b/scripts/._tl.js differ
Binary files a/scripts/._tl.py and b/scripts/._tl.py differ
Binary files a/scripts/._tlcheck.sh and b/scripts/._tlcheck.sh differ
Binary files a/scripts/._toc.sh and b/scripts/._toc.sh differ
Binary files a/scripts/._uncomment.py and b/scripts/._uncomment.py differ
Binary files a/scripts/._ziploader.py and b/scripts/._ziploader.py differ
Binary files a/scripts/bench/._filehash.sh and b/scripts/bench/._filehash.sh differ
Binary files a/scripts/deps-docker/._Dockerfile and b/scripts/deps-docker/._Dockerfile differ
Binary files a/scripts/deps-docker/._Makefile and b/scripts/deps-docker/._Makefile differ
Binary files a/scripts/deps-docker/._busy-mp3.sh and b/scripts/deps-docker/._busy-mp3.sh differ
Binary files a/scripts/deps-docker/._codemirror.patch and b/scripts/deps-docker/._codemirror.patch differ
Binary files a/scripts/deps-docker/._easymde-ln.patch and b/scripts/deps-docker/._easymde-ln.patch differ
Binary files a/scripts/deps-docker/._easymde-marked6.patch and b/scripts/deps-docker/._easymde-marked6.patch differ
Binary files a/scripts/deps-docker/._easymde.patch and b/scripts/deps-docker/._easymde.patch differ
Binary files a/scripts/deps-docker/._genprism.py and b/scripts/deps-docker/._genprism.py differ
Binary files a/scripts/deps-docker/._genprism.sh and b/scripts/deps-docker/._genprism.sh differ
Binary files a/scripts/deps-docker/._markdown-it.patch and b/scripts/deps-docker/._markdown-it.patch differ
Binary files a/scripts/deps-docker/._marked-ln.patch and b/scripts/deps-docker/._marked-ln.patch differ
Binary files a/scripts/deps-docker/._marked.patch and b/scripts/deps-docker/._marked.patch differ
Binary files a/scripts/deps-docker/._mini-fa.css and b/scripts/deps-docker/._mini-fa.css differ
Binary files a/scripts/deps-docker/._mini-fa.sh and b/scripts/deps-docker/._mini-fa.sh differ
Binary files a/scripts/deps-docker/._shiftbase.py and b/scripts/deps-docker/._shiftbase.py differ
Binary files a/scripts/deps-docker/._showdown.patch and b/scripts/deps-docker/._showdown.patch differ
Binary files a/scripts/deps-docker/._zopfli.makefile and b/scripts/deps-docker/._zopfli.makefile differ
Binary files a/scripts/docker/._Dockerfile.ac and b/scripts/docker/._Dockerfile.ac differ
Binary files a/scripts/docker/._Dockerfile.dj and b/scripts/docker/._Dockerfile.dj differ
Binary files a/scripts/docker/._Dockerfile.djd and b/scripts/docker/._Dockerfile.djd differ
Binary files a/scripts/docker/._Dockerfile.djf and b/scripts/docker/._Dockerfile.djf differ
Binary files a/scripts/docker/._Dockerfile.djff and b/scripts/docker/._Dockerfile.djff differ
Binary files a/scripts/docker/._Dockerfile.dju and b/scripts/docker/._Dockerfile.dju differ
Binary files a/scripts/docker/._Dockerfile.im and b/scripts/docker/._Dockerfile.im differ
Binary files a/scripts/docker/._Dockerfile.iv and b/scripts/docker/._Dockerfile.iv differ
Binary files a/scripts/docker/._Dockerfile.min and b/scripts/docker/._Dockerfile.min differ
Binary files a/scripts/docker/._Dockerfile.min.pip and b/scripts/docker/._Dockerfile.min.pip differ
Binary files a/scripts/docker/._Makefile and b/scripts/docker/._Makefile differ
Binary files a/scripts/docker/._README.md and b/scripts/docker/._README.md differ
Binary files a/scripts/docker/._devnotes.md and b/scripts/docker/._devnotes.md differ
Binary files a/scripts/docker/._innvikler.sh and b/scripts/docker/._innvikler.sh differ
Binary files a/scripts/docker/._make.sh and b/scripts/docker/._make.sh differ
Binary files a/scripts/py2/._queue and b/scripts/py2/._queue differ
Binary files a/scripts/py2/queue/.___init__.py and b/scripts/py2/queue/.___init__.py differ
Binary files a/scripts/pyinstaller/._README.md and b/scripts/pyinstaller/._README.md differ
Binary files a/scripts/pyinstaller/._build.sh and b/scripts/pyinstaller/._build.sh differ
Binary files a/scripts/pyinstaller/._depchk.sh and b/scripts/pyinstaller/._depchk.sh differ
Binary files a/scripts/pyinstaller/._deps.sha512 and b/scripts/pyinstaller/._deps.sha512 differ
Binary files a/scripts/pyinstaller/._deps.txt and b/scripts/pyinstaller/._deps.txt differ
Binary files a/scripts/pyinstaller/._ffmpeg.patch and b/scripts/pyinstaller/._ffmpeg.patch differ
Binary files a/scripts/pyinstaller/._ffmpeg.txt and b/scripts/pyinstaller/._ffmpeg.txt differ
Binary files a/scripts/pyinstaller/._icon.sh and b/scripts/pyinstaller/._icon.sh differ
Binary files a/scripts/pyinstaller/._loader.py and b/scripts/pyinstaller/._loader.py differ
Binary files a/scripts/pyinstaller/._loader.rc and b/scripts/pyinstaller/._loader.rc differ
Binary files a/scripts/pyinstaller/._notes.txt and b/scripts/pyinstaller/._notes.txt differ
Binary files a/scripts/pyinstaller/._up2k.rc and b/scripts/pyinstaller/._up2k.rc differ
Binary files a/scripts/pyinstaller/._up2k.sh and b/scripts/pyinstaller/._up2k.sh differ
Binary files a/scripts/pyinstaller/._up2k.spec and b/scripts/pyinstaller/._up2k.spec differ
Binary files a/scripts/pyinstaller/._up2k.spec.sh and b/scripts/pyinstaller/._up2k.spec.sh differ
Binary files a/scripts/strip_hints/._a.py and b/scripts/strip_hints/._a.py differ
Binary files a/scripts/test/._ptrav.py and b/scripts/test/._ptrav.py differ
Binary files a/scripts/test/._race.py and b/scripts/test/._race.py differ
Binary files a/scripts/test/._smoketest.py and b/scripts/test/._smoketest.py differ
Binary files a/scripts/test/._tftp.sh and b/scripts/test/._tftp.sh differ
Binary files a/srv/._ceditable.html and b/srv/._ceditable.html differ
Binary files a/srv/._chat.md and b/srv/._chat.md differ
Binary files a/srv/._expand and b/srv/._expand differ
Binary files a/srv/._extend.md and b/srv/._extend.md differ
Binary files a/srv/._test.md and b/srv/._test.md differ
Binary files a/srv/expand/._README.md and b/srv/expand/._README.md differ
Binary files a/tests/.___init__.py and b/tests/.___init__.py differ
Binary files a/tests/._ptrav.py and b/tests/._ptrav.py differ
Binary files a/tests/._res and b/tests/._res differ
Binary files a/tests/._run.py and b/tests/._run.py differ
Binary files a/tests/._test_cp.py and b/tests/._test_cp.py differ
Binary files a/tests/._test_dedup.py and b/tests/._test_dedup.py differ
Binary files a/tests/._test_dots.py and b/tests/._test_dots.py differ
Binary files a/tests/._test_dxml.py and b/tests/._test_dxml.py differ
Binary files a/tests/._test_hooks.py and b/tests/._test_hooks.py differ
Binary files a/tests/._test_httpcli.py and b/tests/._test_httpcli.py differ
Binary files a/tests/._test_idp.py and b/tests/._test_idp.py differ
Binary files a/tests/._test_metrics.py and b/tests/._test_metrics.py differ
Binary files a/tests/._test_mv.py and b/tests/._test_mv.py differ
Binary files a/tests/._test_utils.py and b/tests/._test_utils.py differ
Binary files a/tests/._test_vfs.py and b/tests/._test_vfs.py differ
Binary files a/tests/._test_webdav.py and b/tests/._test_webdav.py differ
Binary files a/tests/._util.py and b/tests/._util.py differ
Binary files a/tests/res/._idp and b/tests/res/._idp differ
Binary files a/tests/res/idp/._1.conf and b/tests/res/idp/._1.conf differ
Binary files a/tests/res/idp/._2.conf and b/tests/res/idp/._2.conf differ
Binary files a/tests/res/idp/._3.conf and b/tests/res/idp/._3.conf differ
Binary files a/tests/res/idp/._4.conf and b/tests/res/idp/._4.conf differ
Binary files a/tests/res/idp/._5.conf and b/tests/res/idp/._5.conf differ
Binary files a/tests/res/idp/._6.conf and b/tests/res/idp/._6.conf differ
diff -ruN -x .git -x __pycache__ -x *.pyc a/tests/util.py b/tests/util.py
--- a/tests/util.py	2026-09-19 21:53:28
+++ b/tests/util.py	2026-09-19 22:49:00
@@ -274,6 +274,12 @@
         self.g403 = Garda("")
         self.gurl = Garda("")
 
+        # Initialize missing attributes that HttpCli accesses
+        import re
+        self.ptn_cc = re.compile(r"[\x00-\x1f]")
+        self.ptn_hsafe = re.compile(r"[\x00-\x1f<>\"'&]")
+        self.uparam_cc_ok = set("doc move tree".split())
+
         self.u2idx = None
 
     def cachebuster(self):
